### PR TITLE
Review content

### PR DESCRIPTION
**Pull request by mariangz**
~~ https://github.com/neighbourhoodie/nh-systemd/blob/793b00273af8b7cc2958e590c26a44ca53167120/docs/_contributing/CODE_QUALITY.md#L74 --> 404
- https://github.com/neighbourhoodie/nh-systemd/blob/793b00273af8b7cc2958e590c26a44ca53167120/docs/_contributing/SECURITY.md#L14  --> I am not sure if this link is correct. It does not submit a subscription request? ~~

:heavy_check_mark:

### CODE REVIEW

File: docs/_concepts/RANDOM_SEEDS.md
Comment (mariangz) [inline docs/_concepts/RANDOM_SEEDS.md:1]: this link doesn't work `[boot loader entropy seed
   support](https://netbsd.gw.com/cgi-bin/man-cgi?boot+8)` line `353`

File: docs/_devdocs/AUTOPGKTEST.md
Comment (mariangz) [inline docs/_devdocs/AUTOPGKTEST.md:1]: This link doesn't work, and I didn't find a new one to replace it `[configured webhooks](https://github.com/systemd/systemd/settings/hooks)` line `29`

File: docs/_devdocs/CONTROL_GROUP_INTERFACE.md
Comment (mariangz) [inline docs/_devdocs/CONTROL_GROUP_INTERFACE.md:1]: - This link ultimately leads back to this same page [Pax Control Groups](http://www.freedesktop.org/wiki/Software/systemd/PaxControlGroups/), so I don't know if we should keep it or not. lines `12` and `49`
- Link line `19` was replaced by this one, not sure if it's the same, I think so [Linux Foundation News Story](https://www.linuxfoundation.jp/blog/2013/08/all-about-the-linux-kernel-cgroups-redesign/)

File: docs/_devdocs/INHIBITOR_LOCKS.md
Comment (mariangz) [inline docs/_devdocs/INHIBITOR_LOCKS.md:1]: I couldn't replace this link [inhibitor API](http://people.gnome.org/~mccann/gnome-session/docs/gnome-session.html#org.gnome.SessionManager.Inhibit) line `160`

File: docs/_devdocs/MINIMAL_BUILDS.md
Comment (mariangz) [inline docs/_devdocs/MINIMAL_BUILDS.md:1]: It's not a 'missing link', but not working currently

File: docs/_devdocs/THE_CASE_FOR_THE_USR_MERGE.md
Comment (mariangz) [inline docs/_devdocs/THE_CASE_FOR_THE_USR_MERGE.md:1]: line `115` this link doesn't work anymore `[mail from Lennart](http://thread.gmane.org/gmane.linux.redhat.fedora.devel/155511/focus=155792)`

File: docs/_interfaces/PORTABILITY_AND_STABILITY.md
Comment (mariangz) [inline docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: these links don't work:
- gentoo: lines 83, 84, 85
- lines: 95, 100, 101, 127

File: docs/_userdocs/FAQ.md
Comment (mariangz) [inline docs/_userdocs/FAQ.md:1]: this link doesn't work, line 73 [logind.conf(7)](http://www.freedesktop.org/software/systemd/man/logind.html)

File: man/systemd-vmspawn.xml [hunk 1]
```diff
@@ -42,7 +42,8 @@
     file descriptors must be passed with the names <literal>kvm</literal> and <literal>vhost-vsock</literal>
     respectively.</para>
 
-    <para>Note: on Ubuntu/Debian derivatives systemd-vmspawn requires the user to be in the <literal>kvm</literal> group to use the VSock options.</para>
+    <para>Note: on Ubuntu/Debian derivatives systemd-vmspawn requires the user to be in the
```
Comment (hulkoba) [inline man/systemd-vmspawn.xml]: This file should not be part of this PR right?

File: docs/_concepts/RANDOM_SEEDS.md [hunk 2]
```diff
@@ -337,72 +322,68 @@ This primarily leaves two kind of systems in the cold:
...
+   throughput there's little it gets us over just using `getrandom()`.
+
+   Also, it's not supported by glibc.
+   And as long as that's the case we are not keen on using it, as we'd have to maintain that on our own, and we don't want to maintain our own cryptographic primitives if we don't have to.
+   Since systemd's uses are not performance relevant (besides the pool initialization
+   delay, which this doesn't solve), there's hence little benefit for us to call these functions.
+   That said, if glibc learns these APIs one day, we'll certainly make use of them where appropriate.
+
+7. *This is boring: NetBSD had
+8. [boot loader entropy seed support](https://netbsd.gw.com/cgi-bin/man-cgi?boot+8) since ages!*
```
Comment (hulkoba) [inline docs/_concepts/RANDOM_SEEDS.md]: oops

File: docs/_concepts/RANDOM_SEEDS.md
Comment (hulkoba) [reply docs/_concepts/RANDOM_SEEDS.md:1]: could it be this https://man.netbsd.org/entropy.7 ?

File: docs/_devdocs/AUTOPGKTEST.md
Comment (hulkoba) [reply docs/_devdocs/AUTOPGKTEST.md:1]: all the line numbers do not work anymore. you should rather copy the line from github

Comment (hulkoba) [reply docs/_devdocs/AUTOPGKTEST.md:1]: This link works. you just have to be logged in as part of systemd team

File: docs/_devdocs/CONTROL_GROUP_INTERFACE.md
Comment (hulkoba) [reply docs/_devdocs/CONTROL_GROUP_INTERFACE.md:1]: everything seems to be ok on this page

File: docs/_devdocs/INHIBITOR_LOCKS.md
Comment (hulkoba) [reply docs/_devdocs/INHIBITOR_LOCKS.md:1]: https://man.archlinux.org/man/extra/gnome-session/gnome-session-inhibit.1.en  ?

File: docs/_devdocs/MINIMAL_BUILDS.md
Comment (hulkoba) [reply docs/_devdocs/MINIMAL_BUILDS.md:1]: where?

File: docs/_concepts/RANDOM_SEEDS.md
Comment (hulkoba) [reply docs/_concepts/RANDOM_SEEDS.md:1]: I would put that link as a suggestion to the PR we open

Comment (hulkoba) [reply docs/_concepts/RANDOM_SEEDS.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_concepts/RANDOM_SEEDS.md#L337

File: docs/_devdocs/INHIBITOR_LOCKS.md
Comment (hulkoba) [reply docs/_devdocs/INHIBITOR_LOCKS.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_devdocs/INHIBITOR_LOCKS.md#L203

File: docs/_devdocs/MINIMAL_BUILDS.md
Comment (hulkoba) [reply docs/_devdocs/MINIMAL_BUILDS.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_devdocs/MINIMAL_BUILDS.md#L26

File: docs/_devdocs/THE_CASE_FOR_THE_USR_MERGE.md
Comment (hulkoba) [reply docs/_devdocs/THE_CASE_FOR_THE_USR_MERGE.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_devdocs/THE_CASE_FOR_THE_USR_MERGE.md#L114

File: docs/_interfaces/PORTABILITY_AND_STABILITY.md
Comment (hulkoba) [reply docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_interfaces/PORTABILITY_AND_STABILITY.md#L82

gentoo: https://wiki.gentoo.org/wiki/Project:GNOME

Comment (hulkoba) [reply docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: sd-messages api https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_interfaces/PORTABILITY_AND_STABILITY.md#L99

Comment (hulkoba) [reply docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: [sd-gpt.h API](https://www.freedesktop.org/software/systemd/man/sd-gpt.html) at https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_interfaces/PORTABILITY_AND_STABILITY.md#L94

Comment (hulkoba) [reply docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_interfaces/PORTABILITY_AND_STABILITY.md#L100
[sd-path.h API](https://www.freedesktop.org/software/systemd/man/sd-path.html)

Comment (hulkoba) [reply docs/_interfaces/PORTABILITY_AND_STABILITY.md:1]: [/etc/timezone](https://www.freedesktop.org/software/systemd/man/timezone.html)

https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_interfaces/PORTABILITY_AND_STABILITY.md#L126

File: docs/_userdocs/FAQ.md
Comment (hulkoba) [reply docs/_userdocs/FAQ.md:1]: https://github.com/neighbourhoodie/nh-systemd/blob/a875c1bdb995ca9cba49474a4605c558daf90254/docs/_userdocs/FAQ.md#L85

[logind.conf(7)](http://www.freedesktop.org/software/systemd/man/logind.html)

**Comment by mariangz** [comment]
The following external links don't work: 
- [https://github.com/systemd/systemd/security/code-scanning](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-75e87309728b1d0f8e4287595f0f7e243f2f260c58bf7a3d9f214b88efd53fc3L74)
- [https://netbsd.gw.com/cgi-bin/man-cgi?boot+8](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-f1d6f9ecd3ae7e019c04421d4b4cca002a002c6a06f1db92bd1b5a40619d538dR338)
- [https://github.com/systemd/systemd/settings/hooks](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-4de746c17ad532bb03f35588c24ae7f3e12bb7811439b434d4e89538d565f3e6R39)
- [http://people.gnome.org/~mccann/gnome-session/docs/gnome-session.html#org.gnome.SessionManager.Inhibit](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-3018e6636a43c394086986a0ccbb788fc9f17aa80c970e3ebf1b99209efc1fa2R204)
- [http://thread.gmane.org/gmane.linux.redhat.fedora.devel/155511/focus=155792](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-04a31922a34467d0d41233a0f95549d6618d8297e4ced25cac1cd252276231fdR115)
- [http://www.gentoo.org/proj/en/desktop/gnome/openrc-settingsd.xml](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-9f294c205b388d14ccf8f20b554d41c63c7742f1a35b0dcc358de66882c2b557R115)
- [https://www.freedesktop.org/software/systemd/man/sd-messages.html](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-9f294c205b388d14ccf8f20b554d41c63c7742f1a35b0dcc358de66882c2b557R132)
- [https://www.freedesktop.org/software/systemd/man/timezone.html](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-9f294c205b388d14ccf8f20b554d41c63c7742f1a35b0dcc358de66882c2b557R159)
- [http://www.freedesktop.org/software/systemd/man/logind.html](https://github.com/neighbourhoodie/nh-systemd/pull/4/files#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17)
